andela/ah-orcas-frontend
# feat(pagination): add pagination support

#### What does this PR do?
```Add pagination support ```
#### Description of Task to be completed?
```Add pagination to our articles to make article traversal easier. ```
#### How should this be manually tested?
- Clone this repository
`git clone https://github.com/andela/ah-orcas-frontend.git`
- Change the working directory to the project folder
`cd ah-orcas-frontend`
- Checkout to the appropriate branch
`git checkout ft-pagination-161254624`
- Install the required dependencies:
     `npm start`
- you should be able to see prev and next button at the bottom of the page.

#### What are the relevant pivotal tracker stories?
[#161254624](https://www.pivotaltracker.com/story/show/161254624)
#### Screenshots (if appropriate)
<img width="1327" alt="screenshot 2019-01-03 at 15 11 52" src="https://user-images.githubusercontent.com/8406752/50637162-f18c5780-0f69-11e9-860e-3e5547b4de9a.png">


#### Checklist:
